shared/appengine.go

```diff
@@ -42,18 +46,38 @@ func (c *clientsImpl) Init(ctx context.Context) (err error) {
 		return nil
 	}
 	c.cloudtasks, err = cloudtasks.NewClient(ctx)
-	return err
+	if err != nil {
+		return err
+	}
+
+	c.gclog, err = gclog.NewClient(ctx, runtimeIdentity.AppID)
```

Comment:
Project ID is different between flex and standard

Reply:
Weird, `GOOGLE_CLOUD_PROJECT` should be available on Flex, too, as well as `GAE_*`: https://cloud.google.com/appengine/docs/flexible/go/migrating

Reply:
Well I was guessing that it could be the cause for searchcache failure. It seems to [retrieve/verify project ID from metadata](https://github.com/web-platform-tests/wpt.fyi/blob/master/api/query/cache/service/main.go#L115). Might be the same underlyingly?